src/clojisr/v1/r.clj

```diff
@@ -49,7 +51,11 @@
 (def clj->java->r (comp java->r clj->java))
 (def clj->r clj->java->r)
 
-(defn r->java->clj [r-object] (-> r-object r r->java java2clj/java->clj))
+(defn r->java->clj [r-object & options] 
+  (-> r-object 
+      (r options) 
+      (r->java options) 
+      (java2clj/java->clj options)))
 (def r->clj r->java->clj)
 
 (defn r->java->native-clj [r-object] (-> r r-object r->java java2clj/java->native))
```

Comment:
I did not change this function to take `options`.
Could not find any usage, neither sucessfully call it myself.

Reply:
We need to pass options only to `java->clj` since this is the place where actual conversion is done.